src/main/webapp/js/userMapData.json

```diff
@@ -1,6 +1,9 @@
 {
-  "lastUpdated": "31 January 2017",
+  "lastUpdated": "15 March 2017",
   "institutes": {
+  	 "Abu Dhabi": [ 	
+  	   "Khalifa University"
+  	  ],
```

Comment:
You are right, it is already in the UAE, so I think its better to only included there.